jasmine/index.html

```diff
@@ -0,0 +1,305 @@
+					<pre><code>describe("a self-driving car", function(){
+  describe("turning in traffic", function(){
+    beforeEach(function(){
+      startEngine();
+      navigate("Safeway");
+    });
+
+    afterEach(function(){
+      navigate("Home");
+      killEngine();
+    });
+
+    it("should signal when turning", function(){
+      turnLeft();
+      expect(leftSignalPosition).toEqual("on");
+    });
+  });
+});
+					</code></pre>
+					<p><a href="http://jasmine.github.io/2.0/introduction.html#section-Setup_and_Teardown" target="_blank">more from the docs</a></p>
+				</section>
+				<section>
+					<h3>Exercise time</h3>
+					<br>
+					<p><a href="step1/instructions.html" target="_blank">Click here for directions</a></p>
+					<p>Suggested time: 30 minutes</p>
+				</section>
+
+				<!-- Jasmine Part 2 -->
+				<section>
+					<h3>Control Flow</h3>
+					<h4>Shoulda, coulda, woulda</h4>
+					<p><strong>Control Flow</strong> is the order in which the individual statements, instructions or function calls are executed or evaluated.</p>
+					<br>
+					<p>Because this can get complicated quickly, we'll need to test this.</p>
+				</section>
+				<section>
+					<h3>Spies</h3>
+					<br>
+					<p>Spies can test the control flow of an application by:</p>
+					<br>
+					<ul>
+						<li>stubbing a function</li>
+						<li>tracking calls to a function</li>
+						<li>exposing arguments that a function was called with</li>
+					</ul>
+				</section>
+				<section>
+					<h3>Using Spies</h3>
+					<ul>
+						<li>spies are usually called on Object methods</li>
+						<li>spies have their own matchers such as <code>toHaveBeenCalled()</code>, <code>toHaveBeenCalledWith()</code>, <a href="http://jasmine.github.io/2.0/introduction.html#section-Spies" target="_blank">and more!</a></li>
+					<pre><code>describe("a robot sidewalk sweeper", function(){
+  it("should look left before crossing the street", function(){
+    spyOn(robot, 'lookLeft'); // set up the spy
+    robot.crossStreet(); // execute
+
+    // use matchers to see if the function was called
+    expect(robot.lookLeft).toHaveBeenCalled();
+    expect(robot.lookLeft.calls.count()).toBeGreaterThan(1);
+  });
+});</code></pre>
+				</section>
+				<section>
+					<h3>Tracking Spies</h3>
+					<br>
+					<p>function calls to spies are tracked through the <code>calls</code> property.</p>
+					<pre><code>expect(robot.lookLeft.calls.count()).toBeGreaterThan(1);</code></pre>
+					<br>
+					<p><a href="http://jasmine.github.io/2.0/introduction.html#section-Other_tracking_properties" target="_blank">Read more about it here!</a></p>
+				</section>
+				<section>
+					<h3>Exercise time</h3>
+					<br>
+					<p><a href="step2/instructions.html" target="_blank">Click here for directions</a></p>
+					<p>Suggested time: 30 minutes</p>
```

Comment:
I'm guessing the exercises will take more like 45, but you can monitor and modify after.